src/transformers/models/esm/modeling_esmfold.py

```diff
@@ -323,7 +323,7 @@ def __init__(self, c_in, eps=1e-5):
     def forward(self, x):
         d = x.dtype
         if d is torch.bfloat16 and not is_deepspeed_initialized():
-            with torch.cuda.amp.autocast(enabled=False):
+            with torch.amp.autocast("cuda", enabled=False):
```

Comment:
The device-agnostic `torch.amp` was added in torch 2.4, and we support torch 2.1 -- we'll have to remove this change :)

(source: https://github.com/pytorch/pytorch/releases/tag/v2.4.0)